loader/icd_dispatch.c

```diff
@@ -18,9 +18,52 @@
 
 #include "icd_dispatch.h"
 #include "icd.h"
+#include "cl_icdl.h"
+#include "icd_version.h"
 #include <stdlib.h>
 #include <string.h>
 
+static cl_int
+clGetICDLoaderInfoOCLICD(
+    cl_icdl_info param_name,
+    size_t       param_value_size,
+    void *       param_value,
+    size_t *     param_value_size_ret)
+{
+    static const char  cl_icdl_OCL_VERSION[] = CL_ICDL_OCL_VERSION_STRING;
+    static const char  cl_icdl_VERSION[] = OPENCL_ICD_LOADER_VERSION_STRING;
+    static const char  cl_icdl_NAME[] = CL_ICDL_NAME_STRING;
+    static const char  cl_icdl_VENDOR[] = CL_ICDL_VENDOR_STRING;
```

Comment:
Do we need these local variables here?  If so, do they need to be "static"?

Reply:
Ah, I was originally thinking this would keep around some extra variables after the function returned but it doesn't look like that's the case, sorry for the distraction.